homeassistant/components/neurio_energy/sensor.py

```diff
@@ -17,22 +20,28 @@
 
 CONF_API_SECRET = 'api_secret'
 CONF_SENSOR_ID = 'sensor_id'
+CONF_SENSOR_IP = 'sensor_ip'
 
 ACTIVE_NAME = 'Energy Usage'
 DAILY_NAME = 'Daily Energy Usage'
+GENERATION_NAME = 'Solar Generation'
+GENERATION_DAILY_NAME = 'Generation Total'
+NET_CONSUMPTION = 'Net Consumption'
+JSON_DATASET = 'API Data Retrieval'
 
 ACTIVE_TYPE = 'active'
 DAILY_TYPE = 'daily'
 
 ICON = 'mdi:flash'
 
-MIN_TIME_BETWEEN_DAILY_UPDATES = timedelta(seconds=150)
-MIN_TIME_BETWEEN_ACTIVE_UPDATES = timedelta(seconds=10)
+MIN_TIME_BETWEEN_DAILY_UPDATES = timedelta(seconds=30)
```

Comment:
Please revert this. We don't want to hammer the API

Reply:
No. We're not hammering the API. Per the documentation that is an acceptable request and no longer generates the multiple request errors as the official code.